dbld/rules

```diff
@@ -19,6 +19,7 @@ TARBALL_IMAGE ?= tarball
 DEFAULT_DEB_IMAGE=ubuntu-bionic
 DEFAULT_RPM_IMAGE=centos-7
 DOCKER=docker
+DOCKER_CACHE_NAMESPACE ?= ghcr.io/syslog-ng
```

Comment:
My suggestion would be `DOCKER_REGISTRY`, or `CONTAINER_REGISTRY`, or any combination of these mixed with the word "namespace".

I think it would be easier to understand what this value really is for someone who wants to change this in their fork.